adds missing include in zend_strtod.c (caused build to fail on sparc)

diff -urNp -x '*.orig' php-5.2.17.org/Zend/zend_strtod.c php-5.2.17/Zend/zend_strtod.c
--- php-5.2.17.org/Zend/zend_strtod.c	2011-01-06 23:48:20.000000000 +0100
+++ php-5.2.17/Zend/zend_strtod.c	2021-10-23 19:00:05.466458470 +0200
@@ -93,6 +93,7 @@
 
 #include <zend_operators.h>
 #include <zend_strtod.h>
+#include <stdint.h>
 
 #ifdef ZTS
 #include <TSRM.h>
